examples/kafka/docker-compose.yaml

```diff
@@ -0,0 +1,52 @@
+version: "3.7"
+services:
+
+  proxy:
+    build:
+      context: .
+      dockerfile: Dockerfile-proxy
+    volumes:
+      - ./envoy.yaml:/etc/envoy.yaml
+    networks:
+      envoymesh:
+        aliases:
+          - envoy
+    expose:
+      - "19092"
+      - "8001"
+    ports:
+      - "19092:19092"
+      - "8001:8001"
+
+  zookeeper:
+    image: wurstmeister/zookeeper:3.4.6
```

Comment:
Update to wurstmeister/zookeeper:latest which is Zookeeper 3.4.13

Reply:
I am reluctant to use `latest` tags in general. And the version difference is not very significant so I'd prefer to leave it as is unless you have a strong opinion about this.

Reply:
I just updated my comment to suggest using the official certified Zookeeper image. Agree with avoiding :latest. I tested with this image and :+1: